### DISCUSSION TITLE
Stratis storage support

### DISCUSSION THREAD

**Discussion by alexminder** [saltstack/salt#67393] [Feature Requests]
### Description of Issue
Add support for [Stratis Storage](https://github.com/stratis-storage) (LVM replacement), please.

### Description
Stratis is a solution made by Red Hat with the goal to be a Volume-managing Filesystem like BTRFS and ZFS (with pools and so on).

[stratis-cli](https://github.com/stratis-storage/stratis-cli) written in Python, so it easy to wrap around this code in to module.

**Comment by frogunder** [comment]
@alexminder Thanks for this issue. 

Will label as a feature request.

Thanks.
